pkg/cache/internal/cache_reader.go

```diff
@@ -111,6 +111,10 @@ func (c *CacheReader) List(_ context.Context, out client.ObjectList, opts ...cli
 	listOpts := client.ListOptions{}
 	listOpts.ApplyOptions(opts)
 
+	if listOpts.Continue != "" {
```

Comment:
I agree with this change, however it is technically breaking because using this before would not do anything, now it returns an error. Could you update the title accordingly?

Reply:
Thank you! I've updated the title 🙂